### PR TITLE
adding parasport collection

### PR DESCRIPTION
**Pull request by domfennell**

### CODE REVIEW

**Comment by domfennell** [comment]
[Add Parasport Collection (and make Confluence doc)](https://trello.com/c/8ljAiQtr/3382-add-parasport-collection-and-make-confluence-doc)

File: v2.json [hunk 1]
```diff
@@ -300,6 +300,14 @@
               "definition": "Activity types that fit under the 'Yoga' drop down filter on the Join the Movement virtual timetable",
               "beta:publisher": "Sport England",
               "inScheme": "https://openactive.io/activity-list"
+            },
+            {
+              "type": "beta:ConceptCollection",
+              "id": "https://bpa-parasport.github.io/parasport-openactive/collections/something-different.jsonld",
+              "prefLabel": "Something a little bit different",
```
Comment (civsiv) [inline v2.json:307]: I have a slight hesitation around the `prefLabel` here, just seems a bit wordy. That might be fine, not sure

Comment (domfennell) [reply v2.json:307]: Thanks for the feedback, @civsiv. @nickevansuk, do you have any thoughts on wordiness of the `prefLabel`?

Comment (nickevansuk) [reply v2.json:307]: Assume this matches the branding on their side ("Something a little bit different"), fine if so?

Comment (domfennell) [reply v2.json:307]: I believe so. Thanks for confirming.

Comment (civsiv) [reply v2.json:307]: Approved then!
